content-src/components/Search/Search.jsx

```diff
@@ -144,7 +142,7 @@ export class _Search extends React.PureComponent {
             onClick={this.onSearchHandoffClick}
             tabIndex="-1">
             <div className="fake-textbox" data-l10n-id="newtab-search-box-search-the-web-text" />
-            <input type="search" className="fake-editable" tabIndex="-1" aria-hidden="true" onDrop={this.onSearchHandoffDrop} onPaste={this.onSearchHandoffPaste} />
+              <input type="search" className="fake-editable" tabIndex="-1" aria-hidden="true" onDrop={this.onSearchHandoffDrop} onPaste={this.onSearchHandoffPaste} />
```

Comment:
What's this intent for? This is actually a sibling of the `<div fake-textbox>`

Reply:
A typo!